shared/src/commonTest/kotlin/com/mbta/tid/mbta_app/viewModel/FavoritesViewModelTest.kt

```diff
@@ -0,0 +1,315 @@
+                                            objects.upcomingTrip(
+                                                predictions
+                                                    .getValue(stop1)
+                                                    .getValue(route1)
+                                                    .getValue(0)
+                                            )
+                                        ),
+                                    alertsHere = emptyList(),
+                                    allDataLoaded = true,
+                                    hasSchedulesToday = false,
+                                    alertsDownstream = emptyList(),
+                                    context = RouteCardData.Context.Favorites,
+                                )
+                            ),
+                            globalData,
+                        )
+                    ),
+                    now,
+                ),
+                RouteCardData(
+                    RouteCardData.LineOrRoute.Route(route2),
+                    listOf(
+                        RouteCardData.RouteStopData(
+                            route2,
+                            stop2,
+                            listOf(
+                                RouteCardData.Leaf(
+                                    RouteCardData.LineOrRoute.Route(route2),
+                                    stop2,
+                                    directionId = 1,
+                                    listOf(patterns.getValue(route2).getValue(1)),
+                                    setOf(stop2.id),
+                                    upcomingTrips =
+                                        listOf(
+                                            objects.upcomingTrip(
+                                                predictions
+                                                    .getValue(stop2)
+                                                    .getValue(route2)
+                                                    .getValue(1)
+                                            )
+                                        ),
+                                    alertsHere = emptyList(),
+                                    allDataLoaded = true,
+                                    hasSchedulesToday = false,
+                                    alertsDownstream = emptyList(),
+                                    context = RouteCardData.Context.Favorites,
+                                )
+                            ),
+                            globalData,
+                        )
+                    ),
+                    now,
+                ),
+            )
+        fun FavoritesViewModel.State.tweakStaticDataTime() =
+            this.copy(
+                staticRouteCardData =
+                    this.staticRouteCardData?.map { routeCardData ->
+                        routeCardData.copy(at = staticDataTime)
+                    }
+            )
```

Comment:
question: I'm having trouble parsing at a glance - why is this necessary?

Reply:
The static route cards are created with [the actual current time](https://github.com/mbta/mobile_app/blob/c72f28a4139515de528a7e56e0450b17c5d1f6f2/shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/RouteCardData.kt#L704), rather than with a parameter we can control.

Reply:
Ah, got it. Any reason not to add now defaulting to the current time as a parameter instead? I think that would help generally for testability and for the readability of this particular test.

Reply:
That’s cleaner, yeah.